views/actionPage.html

```diff
@@ -0,0 +1,118 @@
+<!-- Header Section -->
```

Comment:
i assume this is the new page that you have added.Is the registry site localized ? if yes, this new page should support localization for the same set of languages that the original registry site supports.

Reply:
Registry page doesn't support localization.